.github/workflows/docker-and-k8s.yml

```diff
@@ -72,13 +73,13 @@ jobs:
 
       - name: Get checksum of the base image files
         run: |
-          python3 build_package_new.py ${{ inputs.image-type }} --files-checksum > image_files_checksum.txt
+          python3 build_package_new.py "${{ inputs.image-type }}-buster" --files-checksum > image_files_checksum.txt
```

Comment:
shouldn't this also be ${{ matrix.image_distro_name }}"

Reply:
To reduce the build time, we only run test jobs for buster. But if you think it's a good idea, I can update the workflow to also run tests for alpine.

Reply:
Pushed a change to also run tests under Alpine. Will see how long it takes.